### ISSUE TITLE
Arrow-up/down to navigate between chats, but not to edit prev. message

### ISSUE THREAD

**Issue by inlinecoder** [overtake/telegram#268]
In the latest version, when editing of a messages was added, hotkeys were changed in a worse way. Now, instead of a convenient navigation up & down between chats with arrows keys, we have to hold CMD+OPTION+ARROW UP/DOWN.

Let's take into account that editing messages is a less frequent scenario than switching between chats, so I can only see only one reason why that decision was made – 'consistency' with Skype & Slack?

TLDR; 
Arrow up – Switch to chat upwards
Arrow down – Switch to chat downwards
CMD+OPTION+Arrow up – Edit prev. message

How does it sound?
